models/DataObject/ClassDefinition/Data/QuantityValue.php

```diff
@@ -278,7 +534,11 @@ public function getDataForResource($data, $object = null, $params = [])
     public function getDataFromResource($data, $object = null, $params = [])
     {
         if ($data[$this->getName() . '__value'] !== null || $data[$this->getName() . '__unit']) {
-            $value = $data[$this->getName() . '__value'];
+            if (is_numeric($data[$this->getName() . '__value'])) {
```

Comment:
```suggestion
            if (!is_numeric($data[$this->getName() . '__value'])) {
```
Shouldn't it be that If it is not already `numeric`, then must be converted with `$this->toNumeric()` ?